### PR TITLE
add IN and IN w/ SATISFIES examples.

### PR DESCRIPTION
**Pull request by jamiltz**

### CODE REVIEW

File: docs/examples/swift-api-walkthrough/api-walkthrough/ViewController.swift [hunk 1]
```diff
@@ -281,12 +283,42 @@ class ViewController: UIViewController {
...
-
+    func inOperatorSatisfiesQuery() {
+        let query = Query.select(
+                SelectResult.expression(Expression.property("sourceairport"))
+            )
+            .from(DataSource.database(database!))
+            .where(
+                Expression.any("FLIGHT_VAR").in(Expression.property("schedule"))
+                    .satisfies(Expression.variable("FLIGHT_VAR.flight").equalTo("AF103"))
+        )
```
Comment (jamiltz) [inline docs/examples/swift-api-walkthrough/api-walkthrough/ViewController.swift]: @pasin this query doesn't succeed currently. Error message in console is:

```
Invalid variable name 'FLIGHT_VAR.flight'
```

Document schema:

```
["sourceairport": ROR, "id": 9920, "airline": AF, "distance": 3213.192432728316, "stops": 0, "destinationairport": NRT, "schedule": <__NSArrayM 0x618000050500>(
{
    day = 0;
    flight = AF832;
    utc = "21:20:00";
},
{
    day = 0;
    flight = AF043;
    utc = "16:08:00";
})
```

File: docs/examples/swift-api-walkthrough/api-walkthrough/ViewController.swift [hunk 2]
```diff
@@ -259,10 +259,9 @@ class ViewController: UIViewController {
...
     }
     
-    // missing the count
-    // see https://developer.couchbase.com/documentation/server/4.6/n1ql/n1ql-language-reference/selectintro.html
     func groupByQuery() {
         let query = Query.select(
+                SelectResult.expression(Function.count("*")),
                 SelectResult.expression(Expression.property("country")),
                 SelectResult.expression(Expression.property("tz"))
             )
```
Comment (jamiltz) [inline docs/examples/swift-api-walkthrough/api-walkthrough/ViewController.swift]: @pasin also updated this example per [this link](https://github.com/couchbase/couchbase-lite-ios/blob/feature/2.0/Swift/Tests/QueryTest.swift#L378). Noticed the n1ql [example](https://developer.couchbase.com/documentation/server/4.6/n1ql/n1ql-language-reference/selectintro.html#topic_xwp_r3k_dx__group-sort-limit) uses `count("*")` and the CBL unit test `count(1)`. Is there any difference in CBL between `count(1)` and `count("*")`?

File: docs/examples/swift-api-walkthrough/api-walkthrough/ViewController.swift (hunk 1 above)
Comment (pasin) [reply docs/examples/swift-api-walkthrough/api-walkthrough/ViewController.swift]: https://github.com/couchbase/couchbase-lite-ios/issues/1875

Comment (pasin) [reply docs/examples/swift-api-walkthrough/api-walkthrough/ViewController.swift]: @jamiltz The issue was fixed in the feature/2.0 branch.

**Comment by pasin** [comment]
Should I merge?

**Comment by pasin** [comment]
As we moved the sample code to unit test, this PR is not valid anymore. I'm closing it.

**Pull request closed without merging**